modules/subscriptions/views.php

```diff
@@ -727,7 +727,9 @@ function jetpack_blog_subscriptions_init() {
 add_action( 'widgets_init', 'jetpack_blog_subscriptions_init' );
 
 function jetpack_register_subscriptions_block() {
-	jetpack_register_block( 'subscriptions' );
+	if ( class_exists( 'WP_Block_Type_Registry' ) && ! WP_Block_Type_Registry::get_instance()->is_registered( 'jetpack/subscriptions' ) ) {
```

Comment:
What triggered this check you've added in 8f2b9260fadd5b4981c3f5102b391292fc4a8923? I am trying to understand how the block could already be registered?

Reply:
See https://github.com/Automattic/jetpack/pull/11212#issuecomment-461770189